### PR TITLE
Added 2 facts for wine issue #16

### PR DESCRIPTION
**Pull request by grahamplata**

### CODE REVIEW

File: facts.txt [hunk 1]
```diff
@@ -21,5 +21,9 @@
     "Orange": {
     	"The Florida oranges are greener than the California oranges. The warm night climate in Florida causes the chlorophyll pigments to migrate into the orange peel making it more greener.",
     	"You will have to eat 7 cups of cornflakes to get the same amount to fiber you would get from one medium orange."
+    },
+    "Wine": {
+        "Since red wine is fermented together with its skin, it has more antioxidants than white wine which is processed without its skin."
+        "Wine names usually indicate the location or the grape varieties used."
```
Comment (rahuldecoded) [inline facts.txt]: Add comma after the braces.

**Comment by rahuldecoded** [comment]
Update your repo.

**Comment by rahuldecoded** [comment]
Add few more facts

**Comment by grahamplata** [comment]
@rahuldecoded sure.

**Comment by rahuldecoded** [comment]
Would you like to contribute in this repo?

**Pull request merged**